packages/web/components/Modal/CreateInstitutionsModal/index.js

```diff
@@ -0,0 +1,262 @@
+/* eslint-disable no-use-before-define */
+/* eslint-disable no-shadow */
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import PlacesAutocomplete, { geocodeByPlaceId } from 'react-places-autocomplete';
+import { toast } from '../../Toast';
+import {
+	Form,
+	Actions,
+	InputField,
+	InputHiddenField,
+	MaskedInputField,
+	SelectField,
+	RequiredIndicator,
+} from '../../Form';
+import { InputLabel } from '../../Form/styles';
+import { Container, Button, Suggestion, GoogleAddressSugestions } from './styles';
+import { Cell, Row } from '../../Common';
+import { createInstitutions } from '../../../services';
+import { STATES } from '../../../utils/enums/states.enum';
+import { unMask } from '../../../utils/helper';
+
+const InstitutionsForm = ({ form, closeModal, loading }) => {
+	const [loadingPlace, setLoadingPlace] = useState(false);
+	const [addressFields, setAddressFields] = useState(false);
+	const [addressInput, setAddressInput] = useState('');
+
+	const onSelect = async (placeId) => {
+		setLoadingPlace(true);
+		setAddressFields(true);
+		const response = await geocodeByPlaceId(placeId);
+		if (response) {
+			const address = response[0];
+			const state = address.address_components[4]?.short_name;
+			form.setValue('address', address.address_components[1]?.long_name);
+			form.setValue('district', address.address_components[2]?.long_name);
+			form.setValue('zipcode', address.address_components[6]?.long_name);
+			form.setValue('state', { label: state, value: state });
+			form.setValue('city', address.address_components[3]?.long_name);
+			form.setValue('lat', address.geometry.location.lat());
+			form.setValue('lng', address.geometry.location.lng());
+			setLoadingPlace(false);
+		}
+	};
+
+	return (
+		<>
+			<Row>
+				<Cell col={12}>
+					<InputField
+						form={form}
+						label="Nome"
+						name="name"
+						type="text"
+						validation={{ required: true }}
+						variant="gray"
+					/>
+				</Cell>
+			</Row>
+			<Row>
+				<Cell col={4}>
+					<InputField
+						form={form}
+						label="Sigla"
+						name="initials"
+						type="text"
+						validation={{ required: true }}
+						variant="gray"
+					/>
+				</Cell>
+				<Cell col={8}>
+					<MaskedInputField
+						form={form}
+						name="cnpj"
+						label="CNPJ"
+						placeholder=""
+						mask="99.999.999/9999-99"
+						pattern={/^\d{2}\.\d{3}\.\d{3}\/\d{4}\-\d{2}$/} // eslint-disable-line no-useless-escape
+						validation={{ required: true }}
+						variant="gray"
+					/>
+				</Cell>
+			</Row>
+			<Row>
+				<Cell col={12}>
+					<InputLabel>
+						Logradouro <RequiredIndicator />
+					</InputLabel>
+					<PlacesAutocomplete
+						id="placeId"
+						name="place_id"
+						variant="gray"
+						value={addressInput}
+						onChange={(value) => setAddressInput(value)}
+						onSelect={(value, placeId) => onSelect(placeId)}
+					>
+						{({ getInputProps, suggestions, getSuggestionItemProps, loading }) => (
```

Comment:
```suggestion
						{({ getInputProps, suggestions, getSuggestionItemProps, loading: placesLoading }) => (
```

Instead of disabling the rule you can rename this param